Example/Example/Modules/Swift/SwiftViewController.swift

```diff
@@ -1,11 +1,31 @@
 import UIKit
 import Faro
 
+class Posts: Mappable {
+
+    required init(json: AnyObject) {
+
+    }
+
+}
+
 class SwiftViewController: UIViewController {
-  @IBOutlet var label: UILabel!
+    @IBOutlet var label: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let bar = ExampleBar()
```

Comment:
❓A small notice about the naming.

I know we discussed the `Bar`, `serve` and `order`. But at the moment I'm having some doubts about the readability. It looks a bit to abstract to me. Sometimes `Service`, `perform` and `call` looks much more self explanatory to me. And that is something I find extremely important.

Our components should be  usable without any documentation at all, and I'm a bit "scared" that this will not be the case with the current naming...

What are your thoughts on this?